server/src/test/java/org/opensearch/search/aggregations/startree/MetricAggregatorTests.java

```diff
@@ -555,26 +554,36 @@ public QueryBuilder getTermsQueryBuilder() {
         }
 
         public QueryBuilder getRangeQueryBuilder() {
-            return new RangeQueryBuilder(fieldName).from(valueSupplier.get())
+            RangeQueryBuilder qb = new RangeQueryBuilder(fieldName).from(valueSupplier.get())
                 .to(valueSupplier.get())
                 .includeLower(randomBoolean())
                 .includeUpper(randomBoolean());
+            return qb;
         }
 
         public QueryBuilder getBoolQueryBuilder() {
             // MUST only
-            BoolQueryBuilder mustOnly = new BoolQueryBuilder().must(getTermQueryBuilder()).must(getRangeQueryBuilder());
+            BoolQueryBuilder mustOnly = new BoolQueryBuilder().must(getTermQueryBuilder());
+            // Keyword terms are not always supported for range queries.
```

Comment:
nit: // Keyword terms are not ~always~ supported for range queries.

(remove always)